src/mcts/search.cc

```diff
@@ -1234,6 +1234,13 @@ void SearchWorker::ExtendNode(Node* node) {
     }
   }
 
+  // A lonely king cannot mate, so indicate that with restricted bounds.
+  if (board.ours().count() < 2) {
+    node->SetBounds(GameResult::DRAW, GameResult::WHITE_WON);
+  } else if (board.theirs().count() < 2) {
```

Comment:
For those curious, clang11 optimizes this to…
```asm
blsrq   -104(%r14), %rax
```
because the 64-bit board will have a single bit set for the lonely king, and the instruction resets the lowest bit resulting in 0 for branching.